e2e/protractor/suites/actions/copy-move/copy.test.ts

```diff
@@ -246,42 +246,72 @@ describe('Copy content', () => {
       undoCopyFolderWithExistingName(folderExisting, '', destinationPF));
   });
 
-  async function copyFile(fileName: string, location: string = '', destination: string, doBefore?: () => void) {
-    if (doBefore) {
-      await doBefore();
+  async function baseCopy(
+    itemName: string | string[],
+    location: string,
+    destination: string,
+    undo: boolean = false,
+    expectedMsg: string = 'Copied 1 item'
```

Comment:
same here - no need for type when default value is used

Reply:
done